Aries/Aries/Agent.swift

```diff
@@ -18,6 +18,8 @@ public class AriesAgent {
     private var storage: Storage!
     public var events: AriesEvents!
     public var connections: AriesConnections!
+    public var credentials: AdminCredentials!
+    public var proofs: AdminProofs!
```

Comment:
```suggestion
```
Admin submodules should not be directly in the agent class. Path to access submodules should be `agent.admin.proofs` or `agent.admin.credential` instead of `agent.adminProofs`or `agent.adminCredentials`